flask_resty/view.py

```diff
@@ -107,8 +108,62 @@ def validate_request_id(self, data, expected_id):
         if id != expected_id:
             raise ApiError(409, {'code': 'invalid_id.mismatch'})
 
+    def update_spec(self, path, spec, many=False, tag=False, **kwargs):
```

Comment:
but then the client could use `ModelApiSpecMixin` on Api View and vice versa. I understand what you are saying, but I don't know if mixins are the best way for this. My intent was actually to couple the `update_spec` to the actual object being specified. update_spec should change when the class is changed, so it makes sense to have them on the same file